### PR TITLE
Modified lvmvdo check

### PR DESCRIPTION
**Pull request by japokorn**
VDO check was failing due to issue in 'vdostats'.
Modified vdo testing so 'lvs' is used to get data instead

### CODE REVIEW

File: tests/verify-pool-member-vdo.yml [hunk 1]
```diff
@@ -1,33 +1,38 @@
 - name: check VDO
   block:
     - name: get information about VDO deduplication
-      command: "vdostats --verbose {{ storage_test_pool.name }}-vdopool-vpool"
+      command: "lvs --noheading -o vdo_deduplication --select 'lv_name=vdopool' {{ storage_test_pool.name }}"
+      #command: "vdostats --verbose {{ storage_test_pool.name }}-vdopool-vpool"
```
Comment (vojtechtrefny) [inline tests/verify-pool-member-vdo.yml:5]: Do we need the old command here? I would just remove the line.

**Review by vojtechtrefny — APPROVED** [review]
Looks good to me otherwise.

File: tests/verify-pool-member-vdo.yml (hunk 1 above)
Comment (japokorn) [reply tests/verify-pool-member-vdo.yml]: Yup, forgot to remove those lines. Fixed.

File: tests/verify-pool-member-vdo.yml [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
+      command: "lvs --noheading -o vdo_deduplication --select 'lv_name=vdopool' {{ storage_test_pool.name }}"
       register: storage_test_vdo_status
       changed_when: false
 
-    - set_fact:
-        storage_test_vdo_dedupe_re: "{{ ('maximum dedupe queries +: +0$') }}"
-
     - assert:
-        that: "storage_test_vdo_status.stdout is regex(storage_test_vdo_dedupe_re)"
+        that: storage_test_vdo_status.stdout |trim != "enabled"
```
Comment (richm) [inline tests/verify-pool-member-vdo.yml:9]:
```suggestion
        that: storage_test_vdo_status.stdout | trim != "enabled"
```

**Comment by richm** [comment]
This new method will work on el7, el8, el9, and fedora?

**Pull request merged**